buildenv

```diff
@@ -0,0 +1,47 @@
+SHDOC_VERSION="1.0.0"
+export ZOPEN_STABLE_URL="https://github.com/ZOSOpenTools/shdocport.git"
```

Comment:
This is pointing to itself, is there an official shdoc repo?

Reply:
Should probably be https://github.com/reconquest/shdoc.git.

Can you add v1.2 as a ZOPEN_STABLE_TAG